### PR TITLE
Remove unnecessary replace statements in `go.mod`

### PR DESCRIPTION
**Pull request by berndverst**
# Description

Remove unnecessary replace statements in `go.mod`

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -374,12 +374,6 @@ require (
 	sigs.k8s.io/yaml v1.3.0 // indirect
 )
 
-replace k8s.io/client => github.com/kubernetes-client/go v0.0.0-20190928040339-c757968c4c36
-
-replace github.com/eclipse/paho.mqtt.golang => github.com/shivamkm07/paho.mqtt.golang v1.3.6-0.20220106130409-e28a1db639f8
```
Comment (ItalyPaleAle) [inline go.mod:379]: Don't remove this

Comment (berndverst) [reply go.mod:379]: Why is this not upstream already? It is unacceptable to use @shivamkm07's personal fork for so many months. We need to find a different strategy.

Comment (berndverst) [reply go.mod:379]: Latest version is 1.4.1 and we are stuck on this unmaintained 1.3.6 fork.

Comment (berndverst) [reply go.mod:379]: Can you complete work on https://github.com/eclipse/paho.mqtt.golang/pull/578 @shivamkm07 ?

**Review by ItalyPaleAle — APPROVED** [review]

File: go.mod (hunk 1 above)
Comment (shivamkm07) [reply go.mod:379]: @berndverst Updated the PR as requested. Waiting for the maintainers' response. Thanks!

**Pull request merged**